packages/halo-theme/src/shared-styles/checkbox.less

```diff
@@ -2,13 +2,12 @@
   --check-color: @checkbox-checked-color;
   color: @control-text-color;
 
-  &:not([readonly]) {
-    cursor: pointer;
-  }
-
   [part='container'] {
     width: 14px;
     height: 14px;
+    &:not([readonly]) {
```

Comment:
This line makes Readonly will become `cursor: pointer`. Are you sure about this?